ADD: top level for A5 Time Transcenders

Landflier opened the pull request:
NB: The symbols assume that the root directory fo the AutoMOS-chipathon2025 git folder is added to XSCHEM_USER_LIBRARY path. 
I.e: All symobls in the A5_top_level.sch have paths starting with AutoMOS-chipathon2025/...

Landflier committed:
ADD: top level for A5 Time Transcenders

NB: The symbols assume that the root directory fo the
AutoMOS-chipathon2025 git folder is added to XSCHEM_USER_LIBRARY path.
All symobls in the A5_top_level have paths starting with
AutoMOS-chipathon2025/...

Landflier commented:
Hi Luighi, I think you can merge, since the PR shows it is pending a merge into integration

LuighiV commented:
> Hi Luighi, I think you can merge, since the PR shows it is pending a merge into integration

Hi @Landflier,

Indeed, I didn´t merge because it didn't follow the same hierarchy as requested by @AlexMantilla1 in the message and that is being followed by other teams. Could you please adapt it to that structure?
Thanks,
Luighi

Landflier committed:
checkpoint

Landflier committed:
Merge branch 'integration' into A5_Time_Transcenders_top_level

Landflier committed:
CHECKPOINT: changed the top_level to match the desired hierarchy

Landflier committed:
CHECKPOINT: added top level of A5. Errors in pin naming for A5 in padring, fixed now

Landflier commented:
Renamed some of the pins in the padring, since they were placed wrong. The correct ordering for our team should be :
**V_LO, V_RF, V_RFb, V_LOb**.

 I wasn't sure if I should have wired up my team's toplevel myself... I could do it so other people won't have to ;)

![Uploading code_001.png…]()

Sincerely,
Vasil

LuighiV commented:
Hi @Landflier ,

are those changes intentional? 
<img width="349" height="343" alt="image" src="https://github.com/user-attachments/assets/9869e796-fd87-47b8-b8f7-f90113d725b8" />

I think they may not be needed to be part of the PR right?

Landflier committed:
FIX: correct directory structure for A5

Landflier commented:
Hi @LuighiV, I fixed the folder structure to follow the convention you mentioned. Thanks for putting up with my slow-ish replies 

About the _tb files under Chipathon2025_pads, I reverted them to their previous commit. There shouldn't be any other changes to them.

Landflier committed:
EDIT: changed directory structure

Landflier commented:
even more edits
